src/App.vue

```diff
@@ -2,15 +2,48 @@
   <div id="app">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Asap:400,400i,500,600,700&display=swap" />
     <drop-zone ref="dropzone" @files-drop="onFilesDrop">
-      <ScaffoldVuer v-if="url" ref="scaffold" class="vuer" :flatmapAPI="flatmapAPI" :display-u-i="displayUI" :url="url"
-        :help-mode="helpMode" :display-latest-changes="true" :display-minimap="displayMinimap"
-        :display-markers="displayMarkers" :enableOpenMapUI="true" :minimap-settings="minimapSettings"
-        :show-colour-picker="showColourPicker" :render="render" :region="region" :view-u-r-l="viewURL" :format="format"
-        :marker-labels="markerLabels" @open-map="openMap" @on-ready="onReady" @scaffold-selected="onSelected"
-        @scaffold-navigated="onNavigated" @timeChanged="updateCurrentTime" @zinc-object-added="objectAdded"
-        @vue:mounted="viewerMounted"/>
+      <ScaffoldVuer
+        v-if="url"
+        ref="scaffold"
+        class="vuer"
+        :flatmapAPI="flatmapAPI"
+        :display-u-i="displayUI"
+        :url="url"
+        :help-mode="helpMode"
+        :helpModeActiveItem="helpModeActiveItem"
```

Comment:
Can you remove this please? This is not used anywhere with ScaffoldVuer. Thanks.

Reply:
It is used to identify the active tooltip in ScaffoldVuer.